docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -539,8 +589,12 @@ convenience to avoid the need to explicitly load classes in the templates.
 
 TODO: config integration
 
-=== CDI Extension Points
-As a CDI based runtime, {project-name} extensions often make CDI beans available as part of the extension behavior. The link:cdi-reference.html[Build Time Extension Points] describes the `BuildItems` you can make use of in your extension build steps to achieve this.
+=== Contexts and Dependency Injection
+
+==== Extension Points
+As a CDI based runtime, {project-name} extensions often make CDI beans available as part of the extension behavior.
```

Comment:
Add a empty line between the section title and the paragraph. Asciidoctor can be dodgy